ComplexParser.py

```
@@ -0,0 +1,104 @@
+
+        lines = []
+
+        with open(self.__stFile, "r") as f:
+            lines = f.readlines()
+
+        complex_vars = []
+        new_lines = []
+        parsing = False
+        type_declaration = False
+
+        for l in lines:
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)